app/helpers/audits_helper.rb

```diff
@@ -31,11 +32,17 @@ def audit_title(audit)
       when 'Puppet Class'
         (id_to_label audit.audited_changes.keys[0], audit.audited_changes.values[0]).to_s
       else
-        name = audit.auditable_name.blank? ? audit.revision.to_label : audit.auditable_name
-        name += " / #{audit.associated_name}" if audit.associated_id && !audit.associated_name.blank?
+        name = if audit.auditable_name.blank?
+                 revision = audit.revision
+                 revision.try(:to_audit_label) || revision.to_label
```

Comment:
I suppose you added this method to avoid problems in orchestration tests, but not every model has `to_audit_label` method defined, try would fail with `NoMethodError` here, it does not render 500 since there's the rescue below. We should not use exceptions for program flow, I think it's better to first verify the object `respond_to?(:to_audit_label)` or define it in application record (does not feel right).